<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>TableView</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>PlainTextEdit</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actOpen"/>
    <addaction name="separator"/>
    <addaction name="actSave"/>
    <addaction name="actClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuClose">
    <property name="title">
     <string>Close(&amp;C)</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuClose"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actPreviewData"/>
   <addaction name="separator"/>
   <addaction name="actAddRow"/>
   <addaction name="actInsertRow"/>
   <addaction name="actDeleteRow"/>
   <addaction name="separator"/>
   <addaction name="actLeft"/>
   <addaction name="actCenter"/>
   <addaction name="actRight"/>
   <addaction name="separator"/>
   <addaction name="actFontBold"/>
   <addaction name="separator"/>
   <addaction name="actClose"/>
  </widget>
  <action name="actNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="document-new" resource="menu_images.qrc">
     <normaloff>:/images/100.bmp</normaloff>:/images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="icon">
    <iconset theme="document-open" resource="menu_images.qrc">
     <normaloff>:/images/001.GIF</normaloff>:/images/001.GIF</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open a file</string>
   </property>
   <property name="statusTip">
    <string>Open a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actClose">
   <property name="icon">
    <iconset resource="menu_images.qrc">
     <normaloff>:/images/132.bmp</normaloff>:/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close a file</string>
   </property>
  </action>
  <action name="actPreviewData">
   <property name="icon">
    <iconset resource="menu_images.qrc">
     <normaloff>:/icons/export1.bmp</normaloff>:/icons/export1.bmp</iconset>
   </property>
   <property name="text">
    <string>PreviewData</string>
   </property>
  </action>
  <action name="actAddRow">
   <property name="icon">
    <iconset resource="menu_images.qrc">
     <normaloff>:/icons/append.bmp</normaloff>:/icons/append.bmp</iconset>
   </property>
   <property name="text">
    <string>AddRow</string>
   </property>
  </action>
  <action name="actInsertRow">
   <property name="icon">
    <iconset resource="menu_images.qrc">
     <normaloff>:/icons/306.bmp</normaloff>:/icons/306.bmp</iconset>
   </property>
   <property name="text">
    <string>InsertRow</string>
   </property>
  </action>
  <action name="actDeleteRow">
   <property name="icon">
    <iconset resource="menu_images.qrc">
     <normaloff>:/icons/delete.bmp</normaloff>:/icons/delete.bmp</iconset>
   </property>
   <property name="text">
    <string>DeleteRow</string>
   </property>
  </action>
  <action name="actLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="menu_images.qrc">
     <normaloff>:/icons/508.bmp</normaloff>:/icons/508.bmp</iconset>
   </property>
   <property name="text">
    <string>left</string>
   </property>
  </action>
  <action name="actCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="menu_images.qrc">
     <normaloff>:/icons/510.bmp</normaloff>:/icons/510.bmp</iconset>
   </property>
   <property name="text">
    <string>center</string>
   </property>
  </action>
  <action name="actRight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="menu_images.qrc">
     <normaloff>:/icons/512.bmp</normaloff>:/icons/512.bmp</iconset>
   </property>
   <property name="text">
    <string>right</string>
   </property>
  </action>
  <action name="actFontBold">
   <property name="icon">
    <iconset resource="menu_images.qrc">
     <normaloff>:/icons/500.bmp</normaloff>:/icons/500.bmp</iconset>
   </property>
   <property name="text">
    <string>bold</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="menu_images.qrc">
     <normaloff>:/icons/save.bmp</normaloff>:/icons/save.bmp</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>plainTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="menu_images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
